samples/nativeSample/src/main/java/com/inmobi/nativead/sample/recyclerview/RecyclerFeedFragment.java

```diff
@@ -99,9 +100,13 @@ public void onConfigurationChanged(Configuration newConfig) {
 
     private void createStrands() {
         for (int position : mAdPositions) {
-            final InMobiNative nativeStrand = new InMobiNative(getActivity(),
-                    PlacementId.YOUR_PLACEMENT_ID_HERE, new StrandAdListener(position));
-            mStrands.add(nativeStrand);
+            try {
+                final InMobiNative nativeStrand = new InMobiNative(getActivity(),
+                        PlacementId.YOUR_PLACEMENT_ID_HERE, new StrandAdListener(position));
+                mStrands.add(nativeStrand);
+            } catch (SdkNotInitializedException e) {
```

Comment:
`try/catch` block is not necessary for `SdkNotInitializedException` as the intention is to throw exception only for wrong integration. Publishers following the guidelines need not even handle this exception as it will never occur.